eng/common/scripts/Helpers/CommandInvocation-Helpers.ps1

```diff
@@ -0,0 +1,51 @@
+function Invoke-LoggedCommand($Command, $ExecutePath, [switch]$GroupOutput)
+{
+    $pipelineBuild = !!$env:TF_BUILD
+    $startTime = Get-Date
+
+    if($pipelineBuild -and $GroupOutput) {
+        Write-Host "##[group]$Command"
+    } else {
+        Write-Host "> $Command"
+    }
+
+    if($ExecutePath) {
+      Push-Location $ExecutePath
+    }
+
+    try {
+      if ($IsLinux -or $IsMacOs)
+      {
+          sh -c "$Command 2>&1"
```

Comment:
Do you really need to shell out? Can't you just invoke-expression of the command?

Reply:
I'm not sure.   I can't remember if there was a reason I started doing that instead of invoke-expression

Reply:
I replaced this with Invoke-Expression and will monitor for issues.  I suspect it came from trying to fork output and pipe stderr to stdout consistently for linux and windows.  Maybe it was just copilot.